hive_plug_play/database/schema.py

```diff
@@ -1,10 +1,11 @@
 
-DB_VERSION = 1
+DB_VERSION = 2
```

Comment:
Haven't tested this, and need to refactor this into seperate views for each project, and add the project to the ini file...